package.json

```diff
@@ -31,15 +31,15 @@
     "yargs": "^15.3.1"
   },
   "scripts": {
-    "prepublishOnly": "tsc",
+    "prepare": "tsc",
     "test": "electron test --extension=ts --require=ts-node/register --exit --js-flags=--expose_gc",
     "test:ci": "yarn test --reporter=mocha-multi-reporters --reporter-options=configFile=.circleci/mocha-reporter-config.json"
   },
   "files": [
     "README.md",
     "package.json",
-    "main/index.js",
-    "renderer/index.js",
+    "main",
+    "renderer",
```

Comment:
doesn't this also include all the `main/*.ts` files, which aren't actually used?

Reply:
This is `main`, not `src/main`.